src/sidebar/services/test/annotations-exporter-test.js

```diff
@@ -255,4 +257,129 @@ ${formattedNow},http://example.com,My group,Annotation,,John Doe,Annotation text
       );
     });
   });
+
+  describe('buildHTMLExportContent', () => {
+    it('throws error when empty list of annotations is provided', () => {
+      assert.throws(
+        () => exporter.buildHTMLExportContent([]),
+        'No annotations to export',
+      );
+    });
+
+    it('generates HTML content with expected annotations', () => {
+      const isoDate = now.toISOString();
+      const annotations = [
+        {
+          ...baseAnnotation,
+          user: 'acct:jane@localhost',
+          tags: ['foo', 'bar'],
+        },
+        {
+          ...baseAnnotation,
+          ...newReply(),
+          target: targetWithSelectors(
+            quoteSelector('includes <p>HTML</p> tags'),
+            pageSelector(23),
+          ),
+        },
+        {
+          ...baseAnnotation,
+          tags: [],
+          target: targetWithSelectors(pageSelector('iii')),
+        },
+      ];
+
+      const result = exporter.buildHTMLExportContent(annotations, {
+        groupName,
+        now,
+      });
+
+      assert.equal(
+        result,
+        `<html lang="en">
+\t<head>
```

Comment:
This is going to be annoying to update if the format ever changes in an unimportant way. It might be more practical to strip formatting from the actual/expected output and just compare that.

Reply:
> It might be more practical to strip formatting from the actual/expected output and just compare that.

Can you elaborate on this?

Reply:
I was thinking of doing `assert.equal(simplifyFormatting(result), simplifyFormatting(expected))` where `simplifyFormatting` is some suitable function (not literally with that name) that normalizes the arguments to remove unimportant whitespace, eg. by stripping leading / trailing spaces from each line.

Reply:
oh, gotcha. Yeah, I think that could work.